.circleci/config.yml

```diff
@@ -175,13 +171,12 @@ jobs:
       - *cache_save_git
       - *cache_restore_bundler
       - *set_ruby
-      - run:
-          name: Setup Build
-          command: |
-            mkdir -p ~/test-reports
-            brew update # Needed because this lane uses "brew bundle" and CircleCI's brew install is too old for that
```

Comment:
I believe we normally don't need this `brew update`...
https://github.com/fastlane/fastlane/pull/17863

I don't know why it failed here though. If the brew version matters, using a newer macOS image might (or might not) help it 🤔 
https://app.circleci.com/pipelines/github/fastlane/fastlane/2629/workflows/c48b73e5-02a7-4563-bf8f-8e19cbb6e508/jobs/66023